### DISCUSSION TITLE
Search returns No rows found for Text field

### DISCUSSION THREAD

**Discussion by sangroxx** [Budibase/budibase#4272] [Help] [answered]
Hi I am trying to add a search form as per the Docs & the YouTube video.

I have a table called "Trucks" with Truck Number & Truck Owner Text Columns and other Options Column.

On the design page, I am trying to add a search form  as per the Docs & the YouTube video. The options picker and binding works flawlessly. However, for the text fields, the results are displayed only & only when the input EXACTLY matches the data.. 

It displays "No Rows Found" _(Separate question, how do I change this no results found text?)_ 
Is it possible to Search/Filter data with only part of the text?

I already tried using the following operators in the Data Provider -
LIKE
EQUALS
STARTS WITH 
None of them work.

The row is only found, when the text input is exactly as the data in the row for the particular column.

[Loom Screen Recording](https://www.loom.com/share/73fb465ef7ce491fb158440f7572b792)

EDIT: Updated Link.

**Comment by pavleks** [comment, marked as answer]
Hi @sangroxx 

> Is it possible to Search/Filter data with only part of the text?
Starts with works for me 
![chrome-capture](https://user-images.githubusercontent.com/76126966/151962522-dd31649f-0973-4f4e-8cfe-e51e5cfa29cc.gif)
![image](https://user-images.githubusercontent.com/76126966/151963605-f383ded4-46ec-4ce5-858a-bd3953a33122.png)

>  (Separate question, how do I change this no results found text?)
I don't think it is possible at this moment.

**Reply by sangroxx** [reply]
Yes, I did the same. But it is still not working.

No rows found.

**Reply by pavleks** [reply]
> Yes, I did the same. But it is still not working.
> 
> No rows found.

If you want, export the app and share the .txt file. I could try to replicate it to see if I could help you.

**Reply by sangroxx** [reply]
> 
[AJRL-export-1643799358498.txt](https://github.com/Budibase/budibase/files/7985803/AJRL-export-1643799358498.txt)

/trucks screen

**Reply by pavleks** [reply]
Hi @sangroxx 
Managed to make it work by recreating the page. 

![chrome-capture (3)](https://user-images.githubusercontent.com/76126966/152177529-5d3a697d-915e-4bde-8931-8381715e0a7c.gif)
[Updated App-export-1643813716974.txt](https://github.com/Budibase/budibase/files/7987548/Updated.App-export-1643813716974.txt)
